cmd/client/lib/cli.go

```diff
@@ -58,9 +66,8 @@ var (
 	}
 	// RelayFlag is the CLI flag for relay peer multiaddr(s) to connect with.
 	RelayFlag = &cli.StringSliceFlag{
-		Name:    "relay",
-		Usage:   "relay peer multiaddr(s) to connect with",
-		Aliases: []string{"peer-with"}, // DEPRECATED
```

Comment:
this was here, i think, because the deploy scripts are using this flag currently. check with @hsanjuan  to confirm migration to "-relay" before removing

Reply:
I believe they are not using this funcationality:

https://github.com/drand/drand-infra/blob/master/packer/relay-gossip.opts.dev
https://github.com/drand/drand-infra/blob/master/packer/relay-gossip.opts.testnet

However, a `-peer-with` flag exists separately again since the `-relay` flag causes the client to create a libp2p node which is unneccessary when `run`-ing a gossip relay since one is already created to publish the randomness to. Hope that makes sense!